server/src/services/cronSchedule/cronSchedule.js

```diff
@@ -35,6 +35,29 @@ const getEmbedIDDocVStep = (steps) => {
   });
 };
 
+/**
+ * // TODO: Document what this is doing and why (including clarifying what the unusual txId2 property is). And refactor / rename this function to be more appropriate.
+ *
```

Comment:
Create a document verification transaction and update the transaction ID for the applicant.

Reply:
Description for some fields of the applicant
txId1 - Identity verification transaction ID
txId2 - Document verification transaction ID
txRecordId1 - Transaction Record ID (TRID) for Identity verification
txRecordId2 - Transaction Record ID (TRID) for Document verification
feTxId - Transaction ID for EmbedID (DocV) - experienceTransactionId

Reply:
@fedencer Thanks for all of your comments! We will incorporate them into code soon.

I'm still confused about the names of these fields:

- txId1
- txId2
- txRecordId1
- txRecordId2
- feTxId

Are those official Trulioo field names? If so, can you please show me where they are in the Trulioo docs?

If not, I think I'd like to rename them.

Would it be appropriate to call them something like txId1 becomes "identityVerificationTransactionId", txRecordId1 becomes "identityVerificationTransactionRecordId", and so on?

And can you please explain the difference between Transaction Record ID and Transaction ID?

Thanks!

cc @sandoche

Reply:
Thanks for clarifying @fedencer

Reply:
No, the database field names were chosen arbitrarily, they are not official Trulioo field names. Of course, it would be better to give them more meaningful names.

A Transaction Record is a derivative document associated with a Trulio transaction. It contains the result of the validation after the transaction receives the completed status. Both identifiers are used in the Trulioo admin interface, so earlier I decided that it is worth storing both identifiers in the database.